### PR TITLE
Sign up

### PR DESCRIPTION
**Pull request by IvayloSlavchev**

### CODE REVIEW

**Review by petya0111 — CHANGES_REQUESTED** [review]
...

File: client/src/components/SignUp/SignUpStyles/Mobile-phone-view.scss [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            background-color: white;
+            position: relative;
+            top: 14rem;
+            left: -.3rem;
+            width: 1rem;
+        }
+    }
+
+    .buttons-group {
+        position: relative;
```
Comment (petya0111) [inline client/src/components/SignUp/SignUpStyles/Mobile-phone-view.scss]: why all of the positioned elements are on relative positions? In the mobile view, it doesn't look fine. Try to improve it with grid or something not so hardcoded. 
[https://ibb.co/B4GLqdL](url)

File: client/src/components/SignUp/Inputs.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                    </Box>
+                </form>
+                <Box className='vertical-line'>
+                    <Box className='vertical-line-text'>or</Box>
+                </Box>
+            </Grid>
+            <Grid xs={12} md={12}>
+                <Box className='buttons-group'>
+                    <Button className='facebook-signUp'><FacebookIcon className='facebook-icon' /> Facebook</Button>
+                    <Button className='google-signUp'><GoogleIcon className='google-icon' /> Continue with Google</Button>
```
Comment (petya0111) [inline client/src/components/SignUp/Inputs.js]: I see that something makes these button texts to be uppercase, find out why it happens.

**Review by petya0111 — CHANGES_REQUESTED** [review]

File: client/src/components/SignUp/Inputs.js (hunk 2 above)
Comment (JordanRad) [reply client/src/components/SignUp/Inputs.js]: MUI buttons are always uppercase, I think

File: client/src/components/SignUp/SignUpStyles/Mobile-phone-view.scss (hunk 1 above)
Comment (JordanRad) [reply client/src/components/SignUp/SignUpStyles/Mobile-phone-view.scss]: You can use the default MUI Sign up page and style it a little bit instead . You can refer to this link - https://mui.com/material-ui/getting-started/templates/

File: client/src/components/SignUp/SignUpStyles/Registration-inputs.scss [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    top: -19rem;
+    display: grid;
+    width: 45%;
+    grid-gap: 2rem;
+
+    .MuiButton-root {
+        width: 70%;
+        padding: .7rem;
+        gap: 3rem;
+    }
```
Comment (JordanRad) [inline client/src/components/SignUp/SignUpStyles/Registration-inputs.scss]: Please, do not overwrite MUI classes

**Review by JordanRad — CHANGES_REQUESTED** [review]

File: client/src/components/SignUp/SignUpStyles/Mobile-phone-view.scss (hunk 1 above)
Comment (IvayloSlavchev) [reply client/src/components/SignUp/SignUpStyles/Mobile-phone-view.scss]: Fixed

File: client/src/components/SignUp/Inputs.js (hunk 2 above)
Comment (IvayloSlavchev) [reply client/src/components/SignUp/Inputs.js]: Fixed

File: client/src/components/SignUp/SignUpStyles/Registration-inputs.scss (hunk 3 above)
Comment (IvayloSlavchev) [reply client/src/components/SignUp/SignUpStyles/Registration-inputs.scss]: Fixed

**Review by petya0111 — APPROVED** [review]
Nice responsive sign-up component ✔️

**Review by JordanRad — APPROVED** [review]

**Pull request merged**